src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -90,14 +86,14 @@ public class ExtensionsRunner {
     private static final Logger logger = LogManager.getLogger(ExtensionsRunner.class);
     private static final String NODE_NAME_SETTING = "node.name";
 
-    private String uniqueId;
-    private DiscoveryNode opensearchNode;
+    private static String uniqueId;
```

Comment:
First of many `static` throughout this PR, and I don't think most of these should be static.  While there's a chance this may not make a difference (if each JVM only has one ExtensionsRunner) I really don't think it's needed.

Reply:
The reason that I use many static is to call the method from the new class which I create. I find another way to call these method, and I will delete these static.